plugins/woocommerce/includes/admin/meta-boxes/views/html-order-item.php

```diff
@@ -47,7 +49,16 @@
 
 	<?php do_action( 'woocommerce_admin_order_item_values', $product, $item, absint( $item_id ) ); ?>
 
-	<td class="item_cost" width="1%" data-sort-value="<?php echo esc_attr( $order->get_item_subtotal( $item, false, true ) ); ?>">
+	<?php if ( wc_get_container()->get( CostOfGoodsSoldController::class )->feature_is_enabled() ) : ?>
+		<td class="item_cost" width="1%" data-sort-value="<?php echo esc_attr( $item->get_cogs_value() ); ?>">
+			<div class="view">
+				<?php
+				echo $item->get_cogs_value_html();
```

Comment:
Maybe we should pass this through `wp_kses_post()` or similar just in case. Given the value is filterable, it might contain unsafe HTML.

Reply:
Ok, that makes sense since other parts of the same page are enclosed in `wp_kses_post` already. Done.